docs/.vitepress/versions.ts

```diff
@@ -1,7 +1,58 @@
+import { execSync } from 'node:child_process';
+import * as semver from 'semver';
 import { version } from '../../package.json';
 
-export const currentVersion = `v${version}`;
+function readBranchName(): string {
+  return (
+    execSync('git branch --show-current').toString('utf8').trim() || 'unknown'
+  );
+}
 
+function readOtherLatestReleaseTagNames(): string[] {
+  const currentMajorVersion = semver.major(version);
+  const latestReleaseTagNames = execSync('git tag -l')
+    .toString('utf8')
+    .split('\n')
+    .filter((tag) => semver.valid(tag))
+    .reduce<Record<number, string[]>>((acc, tag) => {
+      const majorVersion = semver.major(tag);
+      // Only consider tags for our deployed website versions,
+      // excluding the current major version.
+      if (majorVersion >= 6 && majorVersion !== currentMajorVersion) {
+        (acc[majorVersion] = acc[majorVersion] || []).push(tag);
```

Comment:
```suggestion
        (acc[majorVersion] ||= []).push(tag);
```

Maybe even `??=` 👀

Reply:
It isn't supported. You told me the same just a few weeks ago.
https://github.com/faker-js/faker/pull/1407#discussion_r985299537
[`09d11f8` (#1407)](https://github.com/faker-js/faker/pull/1407/commits/09d11f8e169b0bf16f0c0105dfc800325afff84f#diff-43806c9d91df883f547b50d72ffc58f389e58df082afa98825ba79217e7bf3ccR17)

Reply:
I can use `??` though.

```suggestion
        (acc[majorVersion] = acc[majorVersion] ?? []).push(tag);
```

Reply:
> It isn't supported. You told me the same just a few weeks ago. [#1407 (comment)](https://github.com/faker-js/faker/pull/1407#discussion_r985299537) [`09d11f8` (#1407)](https://github.com/faker-js/faker/pull/1407/commits/09d11f8e169b0bf16f0c0105dfc800325afff84f#diff-43806c9d91df883f547b50d72ffc58f389e58df082afa98825ba79217e7bf3ccR17)

The small difference here is, that it is s script on our side and doesn't need to be bundled by esbuild for the end-user

Reply:
Netlify seems to support it.